db/seeds.rb

```diff
@@ -58,12 +60,12 @@
   password: "123456",
   first_name: "Jonathan",
   last_name: "Troupe",
-  weight: 52,
+  weight: 62,
   height: 180,
   address: "4875 Dufferin",
   gender: "Male",
   belt: "Black",
-  years_of_experience: 40,
+  years_of_experience: 10,
```

Comment:
I'm offended you don't think I have 40 years of experience, longer then I've been alive :) haha